code/modules/mining/ore_redemption_machine/survey_vendor.dm

```diff
@@ -64,6 +66,11 @@
 		EQUIPMENT("Jump Boots",									/obj/item/clothing/shoes/bhop,										250),
 		EQUIPMENT("Mini-Translocator",							/obj/item/device/perfect_tele/one_beacon,							120),
 		EQUIPMENT("Survival Equipment - Insulated Poncho",		/obj/random/thermalponcho,											75),
+		EQUIPMENT ("Survival Equipment - Glowstick", 			/obj/item/device/flashlight/glowstick,								10),
+		EQUIPMENT ("Survival Equipment - Flare", 				/obj/item/device/flashlight/flare,									20),
+		EQUIPMENT ("Survival Equipment - Bluespace Glowstick", 	/obj/item/device/flashlight/glowstick/bluespace,					40),
+		EQUIPMENT ("Survival Equipment - Explorer Headset", 	/obj/item/device/radio/headset/explorer,							40),
+		EQUIPMENT ("Survival Equipment - Deluxe Explorer Suit",	/obj/item/clothing/suit/armor/pcarrier/explorer/deluxe,				100),
```

Comment:
The upgraded explorer suit is a bit too cheap.

Reply:
it's at 150 now, you do have to add your own amour